crates/tauri/src/ipc/channel.rs

```diff
@@ -154,9 +154,9 @@ impl JavaScriptChannelId {
             ))?;
           }
           InvokeResponseBody::Raw(bytes) if bytes.len() < MAX_RAW_DIRECT_EXECUTE_THRESHOLD => {
+            let formated_bytes = serde_json::to_string(&bytes)?;
```

Comment:
why formated_bytes? isn't `serialized_bytes` a better name, or `bytes_as_json`?

Reply:
Changed name to `bytes_as_json_array`, didn't know what to name it, and `formated_bytes` was the first name popped in my head 🙃